.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,27 @@
+name: Feature Request
+description: File a feature request
+labels: ["enhancement"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Background Work
+      description: Yes, I searched the following areas for a prior feature/solution to this problem.
+      options:
+        - label: Yes, I searched the [mailing list](https://groups.google.com/forum/#!forum/firesim)
+          required: true
+        - label: Yes, I searched the [documentation](https://docs.fires.im/en/stable/)
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Feature Description
+      description: Description of feature wanted
```

```suggestion
      description: Description of the desired feature.
```

Alternatively, `Description of the feature wanted.`